src/google/protobuf/compiler/ruby/ruby_generator.cc

```diff
@@ -527,6 +527,42 @@ bool MaybeEmitDependency(const FileDescriptor* import,
   }
 }
 
+bool GenerateDSLDescriptor(const FileDescriptor* file, io::Printer* printer,
```

Comment:
GenerateDslDescriptor for the google style guide

Reply:
Done.